src/components/Bomberman.tsx

```diff
@@ -166,50 +184,40 @@ export class BombermanGame extends React.Component<{}, GameState> {
   }
 
   setBomb(player: string) {
-    console.log('setting bombs');
-    const newMap = this.updatedMap;
-    const location = newMap.indexOf(player);
-    newMap[location] = 'bomb1';
-    const possibleLocations: number[] = [
-      location + 1,
-      location - 1,
-      location + this.state.width,
-      location - this.state.width,
-    ];
-    let canMove: boolean = false;
-    for (let i = 0; i < 4; i++) {
-      if (this.isAccesible(location, possibleLocations[i])) {
-        canMove = true;
-      }
-    }
-    if (canMove) {
-      let targetLocation = possibleLocations[Math.floor(Math.random() * 4)];
-      while (!this.isAccesible(location, targetLocation)) {
-        targetLocation = possibleLocations[Math.floor(Math.random() * 4)];
-      }
-      newMap[targetLocation] = player;
-      this.updatedMap = newMap;
-    }
+    const location = this.updatedMap.indexOf(player);
+    this.updatedMap[location] = player + 'bomb1';
   }
 
   ageBombs() {
     const explosions = [];
     const updatedMap: string[] = this.updatedMap;
     for (let i = 0; i < this.state.width * this.state.higth; i++) {
-      switch (updatedMap[i]) {
+      if (updatedMap[i] === 'fire') {
+        updatedMap[i] = 'empty';
+      }
+      switch (

```

Comment:
Seems like in general there's quite a bit of repeated logic here (e.g. between `bomb1` + `bomb2`). It'd probably benefit from extracting some functions to contain chunks of the logic (e.g. why is length 5 important?)